### PR TITLE
Add logging for cancel, fulfill, and expire operations

### PR DESCRIPTION
**Pull request by betaredex**

### CODE REVIEW

File: esi_leap/objects/lease.py [hunk 1]
```diff
@@ -127,6 +129,7 @@ def fulfill(self, context=None):
                             external=True)
         def _fulfill_lease():
 
+            LOG.info("Lease %s fulfilling", self.uuid)
```
Comment (tzumainn) [inline esi_leap/objects/lease.py:132]: I'd reword this as 'Fulfilling lease' just for consistency

File: esi_leap/objects/lease.py [hunk 2]
```diff
@@ -138,6 +141,8 @@ def _fulfill_lease():
         _fulfill_lease()
 
     def expire(self, context=None):
+
+        LOG.info("Lease %s expiring", self.uuid)
```
Comment (tzumainn) [inline esi_leap/objects/lease.py:145]: And here. We can probably get rid of the blank line as well.

**Review by tzumainn — COMMENTED** [review]
Looks good! Could you also add similar logging statements to offer.py?

**Comment by betaredex** [comment]
Fixed spacing and wording

**Comment by betaredex** [comment]
> Looks good! Could you also add similar logging statements to offer.py?

Done!

**Review by tzumainn — APPROVED** [review]
Looks good - thanks!
